src/assets/text/modelingText.js

```diff
@@ -59,7 +59,7 @@ export default {
             {
                 'title': "Deep learning (RGCN_ptrn)",
                 'method': "Beyond space and time, We know quite a bit about how and why streams gain or lose heat – we shouldn’t ignore this wealth of knowledge simply because we’re using a data-driven approach. These process are intuitive – warm, sunny days should warm streams, and cold, cloudy days should cool streams.",
-                'flubber_id':'RGDN_ptrn'
+                'flubber_id':'RGCN_ptrn'
```

Comment:
😅 my bad

Reply:
No worries! I seem to forget that acronym every time I go to type it, so I've had lots of typos, too.